tools/CoherenceBuild/CoherenceVerifier.cs

```diff
@@ -104,24 +118,13 @@ private void Visit(PackageInfo packageInfo)
         {
             if (packageInfo.IsPartnerPackage)
             {
-                if ((_verifyBehavior & CoherenceVerifyBehavior.PartnerPackages) != CoherenceVerifyBehavior.PartnerPackages)
```

Comment:
The flag right now is meant to determine if our partner packages are coherent. We've almost never enabled this because typically none of the CoreFx packages are determined to be coherent using just the nuspec metadata. The rids file lines these up which we don't have any code for. Plus it's not really useful for us to verify partner packages.

The change I made now was to use this flag to determine if we want to ignore our packages not being "coherent" with the packages we've mirrored. This is useful for times like now where we've picked up NETStandard.Library 2.0.0-* (it's a transitive dependency of M.NetCore.App), but none of our packages use it. Right now the only way to avoid coherence to stop failing is to disable verification entirely. Instead, we can choose to verify either coherency among our packages (almost never a reason for this to fail or be turned off), and additionally coherency with partner packages.